MSAL/src/public/MSALPublicClientApplication.h

```diff
@@ -324,6 +324,42 @@
                 correlationId:(NSUUID *)correlationId
               completionBlock:(MSALCompletionBlock)completionBlock;
 
+/*!
+ Acquire a token for a new account using interactive authentication
+ 
+ @param  scopes                  Permissions you want included in the access token received
```

Comment:
nit: can you please align description like in other comments :)

Reply:
done :)